_includes/manuals/1.24/3.2.4_tuning_options.md

```diff
@@ -0,0 +1,54 @@
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from foreman environments deployed at scale in large customer environments.
+
+When the foreman-installer is run, it is deployed with a `default` predefined tuning profile. The definition of this profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
+
+Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Based on your environment needs, use one of the tuning profiles in the installer as shown below.
+
+```bash
+foreman-installer --tuning medium
+```
+
+```bash
+foreman-installer --tuning large
+```
+
+```bash
+foreman-installer --tuning extra-large
+```
+
+```bash
+foreman-installer --tuning extra-extra-large
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+**Note** Using the `--tuning` option above does not update `/ect/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile.
+
+Use the following command to identify the current tuning level:
+
+```bash
+cat /etc/foreman-installer/scenarios.d/foreman.yaml | grep 'tuning'
```

Comment:
I'd prefer to recommend `foreman-installer --help | grep tuning`.

Reply:
Does that show the current value?

Reply:
Note the `default:` part.

```console
# foreman-installer --help | grep tuning
    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "default")
```
After changing it to large:
```console
# foreman-installer --help | grep tuning
    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "large")
```

Reply:
oh wait, this totally beats the `default` purpose.  I thought `default` is something which is really `default`, do we really  use it to show the current configured value across the installer?

Reply:
The word "default" is a bit overloaded. In this case `default:` means the default value for the command line parameter if not specified. So `default: "default"` means the default tuning is configured.

Reply:
Updated now